examples/apigatewayv2/domainname.yaml

```diff
@@ -10,7 +10,7 @@ spec:
   forProvider:
     region: us-east-1
     domainNameConfigurations:
-       # Chage this to the ARN of the certificate created by examples/acm.
-    -  certificateARN: arn:aws:acm:us-east-1:609897127049:certificate/6260abcb-f4c1-4fe2-8542-f360aaacf19b
+      # Change this to the ARN of the certificate created by examples/acm.
+    - certificateARN: arn:aws:acm:us-east-1:609897127049:certificate/497c3833-2861-4807-9d8c-5ab825ada3d1
```

Comment:
Is it OK to expose our account ID?

Reply:
It's not ideal, but I'm also not super worried given it's an account we only use for testing Crossplane. It's worth redacting in a future PR.